api/client/examples/examples/model-endpoints/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gojek/merlin/client"
+	"golang.org/x/oauth2/google"
+)
+
+func main() {
```

Comment:
What's the purpose of this source code?

Reply:
ah this one is generated, maybe I need to only pick the necessary files